src/common/gui/theme.ts

```diff
@@ -1,9 +1,7 @@
-import { deviceConfig } from "../misc/DeviceConfig"
-import { assertMainOrNodeBoot, isApp, isDesktop, isTest } from "../api/common/Env"
-import type { HtmlSanitizer } from "../misc/HtmlSanitizer"
-import { NativeThemeFacade, ThemeController, WebThemeFacade } from "./ThemeController"
+import { assertMainOrNodeBoot } from "../api/common/Env"
 import { isColorLight } from "./base/Color"
 import { themes } from "./builtinThemes"
+import { client } from "../misc/ClientDetector.js"
```

Comment:
I feel a bit off about importing it here but I guess it's a general case that the `themeOptions` should be somewhere else